src/WebJobs.Extensions.Http/Config/HttpExtensionOptions.cs

```diff
@@ -13,12 +13,12 @@ namespace Microsoft.Azure.WebJobs.Extensions.Http
     /// <summary>
     /// Defines the configuration options for the Http binding.
     /// </summary>
-    public class HttpExtensionConfiguration : IExtensionConfigProvider
+    public class HttpExtensionOptions
     {
         /// <summary>
         /// Constructs a new instance.
         /// </summary>
-        public HttpExtensionConfiguration()
+        public HttpExtensionOptions()
```

Comment:
Rename to HttpOptions? The pattern for the other extensions I see is <extension name>Options. E.g. ServiceBusOptions, which is what I'd expect. However, I also see JobHostQueuesOptions, JobHostBlobsOptions which I think should be changed. We need a clear consistent pattern here.

Reply:
Kind of like you have HttpHostBuilderExtensions, StorageHostBuilderExtensions, etc.